src/model/mod.rs

```diff
@@ -28,6 +29,9 @@ pub const LINK_TRAIT: &str = "link";
 /// for a manifest
 pub const LATEST_VERSION: &str = "latest";
 
+/// Type Alias for names of components
+type ComponentName = String;
+
```

Comment:
I would prefer just using a `String` in the functions below, especially since it's a public API, and using doc comments to indicate that its a component name

Reply:
Sure!

- [ ] Remove `type` aliases for `String`s